### PR TITLE
Create useIsAuthenticated, hasRole and hasOrgRole hooks

### PR DESCRIPTION
**Pull request by donbernar**
### Refs
- Solves [SLAS-401](https://getdebrief.atlassian.net/browse/SLAS-401)

### What
- [x] Add useIsAuthenticated public hook
- [x] Add hasRole public hook
- [x] Add hasOrgRole public hook

### Why
We want to expose more useful hooks to enhance dev experience

### CODE REVIEW

**Review by nedrocks — APPROVED** [review]

**Comment by nedrocks** [comment]
This looks great!

Can you also bump the package.json version so we can deploy it? Just a patch version is fine.

**Comment by donbernar** [comment]
Yeah, I'll do so! What about public docs or Changelog @nedrocks ?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slashauth/slashauth-react",
-  "version": "0.9.8",
+  "version": "0.10.8",
```
Comment (nedrocks) [inline package.json]: Let's just make this a patch version.

If you do decide to update the minor version, reset patch version to 0 (<MAJOR>.<MINOR>.<PATCH>)

**Comment by nedrocks** [comment]
RE Changelog -- there isn't really one right now. Let's create one! Do you mind creating a file `CHANGELOG.md` pelase? 

It could look something like this: https://gist.github.com/juampynr/4c18214a8eb554084e21d6e288a18a2c

**Review by nedrocks — APPROVED** [review]
This looks good -- can you test it on beta with slashauth-frontend and then go ahead and release when that looks good?

**Pull request merged**